src/services/database.js

```diff
@@ -264,3 +251,69 @@ export async function setTriviaHints(id, hints) {
     throw err;
   }
 }
+
+export async function getTriviaHintLevel() {
+  try {
+    const res = await getClient().query(
+      "SELECT vint FROM status WHERE vname='trivia_hint_level'"
+    );
+    return res.rows[0].vint;
+  } catch (err) {
+    throw err;
+  }
+
+}
+
+export async function increaseTriviaHintLevel() {
+  try {
+    await getClient().query(
+      "UPDATE status SET vint=vint+1 WHERE vname='trivia_hint_level' AND vint<3"
+    );
+
+  } catch (err) {
+    //don't throw
```

Comment:
But maybe log?

Reply:
nah, if vint can't be increased, then not doing anything is acceptable behavior